packages/network/src/config.ts

```diff
@@ -366,22 +58,24 @@ const genUrls = (network: string) => {
     relayer: {
       url: relayerURL(network),
       provider: {
-        url: rpcUrl,
+        url: rpcUrl
       }
     },
     indexerUrl: indexerURL(network)
   }
 }
 
 export const allNetworks = validateAndSortNetworks([
+  {
+    ...networks[ChainId.POLYGON],
+    ...genUrls('polygon'),
+    isDefaultChain: true,
+    isAuthChain: true
```

Comment:
I think we can remove `isAuthChain` in general now with this version..? but, we can perhaps make a separate ticket and remove it separately to keep this PR just on the moving to constants, etc

Reply:
Yeah, unfortunately I think we have to keep this because v1 sequence.js expects this property and at some point the wallet passes these networks to the dapp.

Reply:
makes sense, lets keep it for now, and later we can abstract it in another way